exercises/sublist/example.lua

```diff
@@ -0,0 +1,12 @@
+return function(sublist, list)
+  if #sublist == 0 then return true end
+
+  for i = 1, #list - #sublist + 1 do
+    for j = 1, #sublist do
+      if list[i + j - 1] ~= sublist[j] then break end
+      if j == #sublist then return true end
```

Comment:
Actually that will cause failures by not checking for the last element to be a match.